OracleFMWInfrastructure/README.md

```diff
@@ -4,23 +4,24 @@ This Docker configuration has been used to create the Oracle Fusion Middleware I
 
 The certification of Oracle FMW Infrastructure on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize/tweak, or create from scratch new scripts and Dockerfiles.
 
+The FMW Infrastructure image is created using the  WebLogic 12.2.1.2 FMW Infrastructure installer. If you are interested in building the root WebLogic Infrastructure image use the files provided in `../OracleWebLogic/dockerfiles/12.2.1.2` and build that image. For more information, visit the [OracleWebLogic](../OracleWebLogic) folder's [README](../OracleWebLogic/README.md) file.
+
 
 ## How to build and run
 This project offers a sample Dockerfile and scripts to build a Oracle Fusion Middleware Infrastructue 12cR2 (12.2.1.2) image. To assist in building the image, you can use the [buildDockerImage.sh](dockerfiles/buildDockerImage.sh) script. See below for instructions and usage.
 
 The `buildDockerImage.sh` script is just a utility shell script that takes the version of the image that needs to be built. Expert users are welcome to directly call `docker build` with their prefered set of parameters.
 
+### Building Oracle JDK (Server JRE) base image
+You must first download the Oracle Server JRE binary and drop in folder `../OracleJava/java-8` and build that image. For more information, visit the [OracleJava](../OracleJava) folder's [README](../OracleJava/README.md) file.
 
-### Building the Oracle WebLogic Infrastructure 12.2.1.2 base image
-**IMPORTANT:**If you are building the Oracle WebLogic Infrastructure image you must first download the Oracle WebLogic Infrastructure 12.2.1.2 binary and drop in folder `../OracleWebLogic/dockerfiles/12.2.1.2` and build that image. For more information, visit the [OracleWeblogic](../OracleWebLogic) folder's [README](../OracleWebLogic/README.md) file.
-
-        $ cd ../OracleWebLogic/dockerfiles
-        $ ./buildDockerImage.sh -v 12.2.1.2 -i
+        $ cd ../OracleJava/java-8
+        $ sh build.sh
 
-### Building Oracle FMW Infrastructure Docker  Image
-**IMPORTANT:**The Oracle FMW Infrastructure image extends the Oracle WebLogic 12.2.1.2 Infrastructure install image. You must build the Oracle WebLogic Infrastructure image. See "Building the Oracle WebLogic Infrastructure 12.2.1.2 base image".
+You can also pull the Oracle Server JRE 8 image from [Oracle Container Registry](https://container-registry.oracle.com) or the [Docker Store](https://store.docker.com/images/oracle-serverjre-8).
```

Comment:
Should provide instructions on how to modify the `Dockerfile` to point to the image pulled from Docker Store.